src/wp-includes/theme.php

```diff
@@ -2097,23 +2097,33 @@ function wp_update_custom_css_post( $css, $args = array() ) {
  * It is a better option to use that class and add any RTL styles to the main stylesheet.
  *
  * @since 3.0.0
+ * @since 6.1.0 Added the `$rtl_mode` parameter. Defaults to 'add'.
  *
  * @global array $editor_styles
  *
- * @param array|string $stylesheet Optional. Stylesheet name or array thereof, relative to theme root.
- *                                 Defaults to 'editor-style.css'
+ * @param array|string $stylesheet Optional. Stylesheet name or array thereof, relative to theme root. Defaults to 'editor-style.css'.
+ * @param string $rtl_mode Optional. Whether if the site in RTL mode. Should the RTL style replace the default sheet or be added as well. Set to 'replace' to replace the sheet.
```

Comment:
```suggestion
 * @param array|string $stylesheet Optional. Stylesheet name or array thereof, relative to theme root.
 *                                 Defaults to 'editor-style.css'.
 * @param string       $rtl_mode   Optional. When the site is in RTL mode, whether the RTL stylesheet should replace the default stylesheet, or be added as well.
 *                                 Accepts 'replace' or 'add'. Defaults to 'add'.
```
- Aligned the parameter names and descriptions.
- Updated the description of `$rtl_mode` for clarity and to show the default value.
- Restored `Defaults to 'editor-style.css'.` to its original position on a new line.